lib/build_config.h.in

```diff
@@ -6,7 +6,9 @@
 #ifndef HSE_BUILD_CONFIG_H
 #define HSE_BUILD_CONFIG_H
 
+// clang-format off
 #define BUILD_CONFIG (@build_config@)
+// clang-format on
```

Comment:
why does clang-format need to be disabled here?

Reply:
It wants to pull the `@` symbols away from the `build_config`, which would throw Meson off.

Reply:
https://github.com/hse-project/hse/actions/runs/4166566419/jobs/7211059172#step:4:8